exercises/patterns/printMultiplicationTable/printMultiplicationTable.js

```diff
@@ -21,6 +21,14 @@ let helpers = require('../printHelpers');
  * @param {number} n - The size of the multiplication table to print
  */
 function printMultiplicationTable(n) {
+  for (let i = 0; i < n; i++) {
```

Comment:
Check out this solution. It prints a 2D table, where each cell is equal to `(i*j)`. The `toString()` function converts this number to a string since the `printCountTimes` function requires a string as the first parameter. The `\t` is called an **escape sequence** and it prints tab spaces, which makes the table look much nicer!
```
for (let i = 1; i <= n; i++) {
    for(let j = 1; j <= n; j++){
        helpers.printCountTimes((i*j).toString(), 1);
        helpers.printCountTimes('\t', 1);
    }
    helpers.printNewLine();
}
```